apps/frontend/src/features/community/DevconCommunityPage.tsx

```diff
@@ -21,16 +21,32 @@ import { useRouter } from "next/router";
 import React, { useEffect, useState } from "react";
 import { toast } from "sonner";
 
-export default function DevconCommunityPage() {
+export default function DevconCommunityPage({
+  displayedDashboard,
+  setDisplayedDashboard,
+}: {
+  displayedDashboard: DisplayedDashboard;
+  setDisplayedDashboard: (dashboard: DisplayedDashboard) => void;
+}) {
   const router = useRouter();
   const [leaderboardDetails, setLeaderboardDetails] =
     useState<LeaderboardDetails | null>(null);
   const [leaderboardEntries, setLeaderboardEntries] =
     useState<LeaderboardEntries | null>(null);
+  const [day1Details, setDay1Details] = useState<LeaderboardDetails | null>(
+    null
+  );
+  const [day1Entries, setDay1Entries] = useState<LeaderboardEntries | null>(
+    null
+  );
+  const [day2Details, setDay2Details] = useState<LeaderboardDetails | null>(
+    null
+  );
+  const [day2Entries, setDay2Entries] = useState<LeaderboardEntries | null>(
+    null
+  );
```

Comment:
Not setting up days 3 and 4 because we can do it later?

Reply:
yep im just fucking exhausted man LOL